uber/templates/registration/index_base.html

```diff
@@ -103,19 +103,19 @@
         <thead>
           <tr>
             {% block tableheadings %}
-            <th> <a href="index?order={{ order.group_id }}">Group Name</a> </th>
+            <th> <a href="index?order={{ order.group_id }}&page={{ page }}&search_text={{ search_text|urlencode }}&invalid={{ invalid }}">Group Name</a> </th>
             <th> Promo Code Group </th>
-            <th> <a href="index?order={{ order.last_first }}">Full Name</a> </th>
-            <th> <a href="index?order={{ order.legal_name }}">Name on ID</a> </th>
-            {% if c.PREASSIGNED_BADGE_TYPES %}<th> <a href="index?order={{ order.badge_printed_name }}">Badge Name</a> </th>{% endif %}
-            <th> <a href="index?order={{ order.badge_type }}">Membership Type</a></th>
-            {% if c.NUMBERED_BADGES %}<th> <a href="index?order={{ order.badge_num }}">Badge #</a> </th> {% endif %}
-            <th> <a href="index?order={{ order.ribbon }}">Badge Ribbons</a></th>
-            <th> <a href="index?order={{ order.amount_extra }}">Upgrade</a></th>
-            <th> <a href="index?order={{ order.paid }}">Paid</a> </th>
+            <th> <a href="index?order={{ order.last_first }}&page={{ page }}&search_text={{ search_text|urlencode }}&invalid={{ invalid }}">Full Name</a> </th>
```

Comment:
I can't find the issue corresponding to this, but this fixes the super annoying behavior on the main registration page where clicking a column header to sort by a column would also effectively take you back to page 0 (the blank page we show people to reduce overhead) and screw up your search terms and whether you had invalid badges turned on.